src/order/filter/basic.lean

```diff
@@ -173,23 +173,36 @@ lemma monotone_mem {f : filter α} : monotone (λ s, s ∈ f) :=
 end filter
 
 namespace tactic.interactive
-open tactic interactive
+open tactic interactive interactive.types lean.parser
```

Comment:
Someone else may know better, but perhaps it works to run `setup_tactic_parser` here instead of manually opening these and having the local postfix?